test/OrchardCore.Tests/Apis/Context/SiteContext.cs

```diff
@@ -109,6 +195,42 @@ public void Dispose()
         {
             Client?.Dispose();
         }
+
+        private async Task<IEnumerable<RecipeDescriptor>> GetRecipesAsync(
+            IEnumerable<IFeatureInfo> features,
+            IEnumerable<IRecipeHarvester> recipeHarvesters)
+        {
+            var recipeCollections = await Task.WhenAll(recipeHarvesters.Select(x => x.HarvestRecipesAsync()));
+            var recipes = recipeCollections.SelectMany(x => x)
+                .Where(r => r.IsSetupRecipe == false &&
+                    !r.Tags.Contains("hidden", StringComparer.InvariantCultureIgnoreCase) &&
+                    features.Any(f => r.BasePath.Contains(f.Extension.SubPath, StringComparison.OrdinalIgnoreCase)));
```

Comment:
All this filtering is not necessary, since we're in a test, only the filter for the recipe name.